File: WEB/exam sample/java/catalog-core/src/main/java/catalog/core/model/Project.java
package catalog.core.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;

@Entity
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class Project extends BaseEntity<Long> {
    public String name, description, members;

    @ManyToOne()
    @JoinColumn(name="managerId")
    private SoftwareDeveloper manager;
}

File: WEB/exam sample/dotnet/Exam/Data/Entities/Project.cs
namespace Data.Entities;

public class Project
{
    public int Id { get; set; }
    public string Name { get; set; }
    public string Description { get; set; }
    public string Members { get; set; }
    public int ManagerId { get; set; }
    
    public SoftwareDeveloper Manager { get; set; }
}